cocos/particle/renderer/particle-system-renderer-data.ts

```diff
@@ -68,19 +68,16 @@ export default class ParticleSystemRenderer {
     }
 
     @type(RenderMode)
-    @displayOrder(3)
+    @serializable
     private _renderMode = RenderMode.Billboard;
 
-    @property
-    @displayOrder(4)
+    @serializable
     private _velocityScale = 1;
 
-    @property
-    @displayOrder(5)
+    @serializable
     private _lengthScale = 1;
 
-    @property
-    @displayOrder(6)
+    @serializable
```

Comment:
@caryliu1999 Notable changes

Reply:
> @caryliu1999 Notable changes

Is the order of display on inspector determined by the order of declarations?